core/git_command.py

```diff
@@ -46,7 +46,7 @@
 -- Partially decoded output follows; � denotes decoding errors --
 """
 
-MIN_GIT_VERSION = (2, 16, 0)
+MIN_GIT_VERSION = (2, 19, 0)
```

Comment:
```suggestion
MIN_GIT_VERSION = (2, 18, 0)
```
https://git-scm.com/docs/git-rebase/2.18.0#Documentation/git-rebase.txt---rebase-mergesrebase-cousinsno-rebase-cousins